rasa/core/channels/rasa_chat.py

```diff
@@ -7,11 +7,9 @@
 import jwt
 import jwt.exceptions
 
-from rasa.core import constants
-from rasa.core.channels.channel import InputChannel
+from rasa.core.channels.channel import InputChannel, _decode_jwt
```

Comment:
can please reference functions by module? And in this case the function should also made public